adc/read_vsys/power_status.c

```diff
@@ -49,13 +49,11 @@ int power_voltage(float *voltage_result) {
     adc_fifo_setup(true, false, 0, false, false);
     adc_run(true);
 
-#if CYW43_USES_VSYS_PIN
-    // We seem to read low values from cyw43 sometimes - this seems to fix it
+    // We seem to read low values initially - this seems to fix it
     int ignore_count = PICO_POWER_SAMPLE_COUNT;
```

Comment:
I wonder if it might be worth making a separate `PICO_POWER_IGNORE_SAMPLE_COUNT` define for this, given that it _always_ seems to be needed?